src/components/List.js

```diff
@@ -121,7 +121,7 @@ const List = ({ firestore }) => {
           };
           const sortedList = sortData(data);
           return isLoading ? (
-            <p>loading...</p>
+            <p className="text-center ">Loading your shopping list...</p>
```

Comment:
Little details like this are what add polish to your app. Nice attention to detail!